pkgs/build-support/kernel/make-initrd-ng-tool.nix

```diff
@@ -7,10 +7,11 @@ rustPlatform.buildRustPackage {
   src = ./make-initrd-ng;
   cargoLock.lockFile = ./make-initrd-ng/Cargo.lock;
 
-  nativeBuildInputs = [ makeWrapper ];
+  passthru.updateScript = ./make-initrd-ng/update.sh;
```

Comment:
I don't think that will work if version is not updated.

Reply:
It did for meshcentral: https://github.com/NixOS/nixpkgs/pull/164212

Reply:
Oh, really. Didn't know that.